### DISCUSSION TITLE
Fancontrol not working

### DISCUSSION THREAD

**Discussion by iueowiqj213** [Rem0o/FanControl.Releases#2446] [Q&A]
There is a exclaimation mark beside my percentage and even if i set it to 0% the fan still run at 100%rpm
![image](https://github.com/Rem0o/FanControl.Releases/assets/166496667/97bac48a-ac89-40bb-8bed-eea407195fd3)

**Comment by Rem0o** [comment]
Mind showing what your calibration looks like?

**Reply by iueowiqj213** [reply]
![image](https://github.com/Rem0o/FanControl.Releases/assets/166496667/8500fb3b-f46c-4bc5-b7a5-295c13c1f3ef)
It is hidden after i click the assisted setup

**Reply by Rem0o** [reply]
If you hover your mouse over the rpm value on the control card of fan 1, what speed sensor is paired with it?

**Reply by iueowiqj213** [reply]
![image](https://github.com/Rem0o/FanControl.Releases/assets/166496667/e9382663-1c6e-4009-a513-61fc838cb1ca)

**Reply by iueowiqj213** [reply]
Im able to control my gpu fan but not fan #1
